client/components/setupWizard/steps/RegisterServerStep.js

```diff
@@ -153,6 +163,19 @@ export function RegisterServerStep({ step, title }) {
 						<Item icon='circle'>{t('Register_Server_Standalone_Own_Certificates')}</Item>
 					</Items>
 				</Option>
+
+				<Label text={<>{t('Register_Server_Registered_I_Agree')} <a href='https://rocket.chat/terms'>{t('Terms')}</a> & <a href='https://rocket.chat/privacy'>{t('Privacy_Policy')}</a></>} position='end' className='SetupWizard__RegisterServerStep__PrivacyTerms'>
```

Comment:
@ggazzo Friendly reminder that we need to find a proper way to interpolate translations!

Reply:
This felt pretty terrible doing it.  But the output of t was always a string when I tried to use the typical %s variable substitution.  Which I guess makes sense